### PR TITLE
fixes redone without conflicts

### PR DESCRIPTION
**Pull request by AndriiDovhopolov-okta**
- Fixes for menu items for homepage

Resolves: 
https://oktainc.atlassian.net/browse/OKTA-360151

### CODE REVIEW

File: packages/@okta/vuepress-theme-prose/const/home.const.js [hunk 1]
```diff
@@ -1,109 +1,110 @@
...
-        image: '/img/homepage/stackselectors/angular.png'
-      },
-       {
-        name: 'Vue.js',
-        link: '/code/vue/',
-        image: '/img/homepage/stackselectors/vue.png'
-      },
-    ]
-  },
+  export const tileData = [
```
Comment (oleksandrpravosudko-okta) [inline packages/@okta/vuepress-theme-prose/const/home.const.js:1]: nit: unnecessary indent

**Review by oleksandrpravosudko-okta — APPROVED** [review]

File: packages/@okta/vuepress-theme-prose/const/home.const.js [hunk 2]
```diff
@@ -1,109 +1,110 @@
-export const tileData = [
-  {
-    name: "Web App",
-    categoryLink: "/docs/guides/sign-into-web-app/aspnet/before-you-begin/",
-    items: [
```
Comment (francoislehoux-okta) [inline packages/@okta/vuepress-theme-prose/const/home.const.js:5]: These were moved to the "home" `.md` file and this file was removed. You will have to re-adjust over there.

Comment (AndriiDovhopolov-okta) [reply packages/@okta/vuepress-theme-prose/const/home.const.js:5]: Thanks for pointing out...

Comment (shawyu-okta) [reply packages/@okta/vuepress-theme-prose/const/home.const.js:5]: try pulling again -- you should see https://github.com/okta/okta-developer-docs/commit/7b1fcdcf75c511ce67389d4c0cf9ab0e89e9e6ca as the tip of your `master` branch

File: packages/@okta/vuepress-theme-prose/const/home.const.js [hunk 3]
```diff
@@ -0,0 +1,146 @@
+  export const tileData = [
```
Comment (francoislehoux-okta) [inline packages/@okta/vuepress-theme-prose/const/home.const.js:1]: Make sure this file stays deleted

Comment (francoislehoux-okta) [reply packages/@okta/vuepress-theme-prose/const/home.const.js:1]: Remember to remove this file and I will give you an approval @AndriiDovhopolov-okta

**Review by francoislehoux-okta — APPROVED** [review]

**Comment by francoislehoux-okta** [comment]
LGTM

**Pull request merged**